Userland/Applications/PixelPaint/Tools/MoveTool.h

```diff
@@ -27,17 +33,16 @@ class MoveTool final : public Tool {
 
 private:
     virtual StringView tool_name() const override { return "Move Tool"sv; }
-    ErrorOr<void> update_cached_preview_bitmap(Layer const* layer);
+    ErrorOr<void> update_cached_preview_bitmap(Layer const*);
+    Optional<Corner const> mouse_in_resize_corner(Layer const*, MouseEvent&);
 
     RefPtr<Layer> m_layer_being_moved;
     Gfx::IntPoint m_event_origin;
     Gfx::IntPoint m_layer_origin;
-    Gfx::IntPoint m_new_scaled_layer_location;
-    Gfx::IntSize m_new_layer_size;
+    Gfx::IntRect m_new_layer_rect;
     bool m_scaling { false };
-    bool m_mouse_in_resize_corner { false };
-    bool m_keep_ascept_ratio { false };
-
+    Optional<Corner const> m_mouse_in_resize_corner {};
```

Comment:
This variable name seems awkward to me. Maybe m_resize_corner_of_mouse, or maybe you have an even better variable name idea :^)

Reply:
I changed the `Corner` enum name to `ResizeAnchorLocation` and changed `m_mouse_in_resize_corner` to `m_resize_anchor_location`